profiles/ShehrozIrfan.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/36946330/209469655-979fe365-cf4a-43ea-a4c2-c925018b9e00.png",
```

Comment:
I recommend more of an image like this. What do you think? 🙂

![image](https://user-images.githubusercontent.com/51391473/209481287-b1a4db93-4b43-4b21-af39-60055c71488a.png)

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/209481287-b1a4db93-4b43-4b21-af39-60055c71488a.png",
```

Reply:
Hey @Panquesito7 
Thanks for your feedback, I've added the updates for image.